orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/CloneServerGroupTask.groovy

```diff
@@ -86,11 +86,13 @@ class CloneServerGroupTask extends AbstractCloudProviderAwareTask implements Tas
 
     List<Map<String, Object>> descriptions = []
     // NFLX bakes images in their test account. This rigmarole is to allow the prod account access to that image.
+    Collection<String> targetRegions = operation.region ? [operation.region] :
```

Comment:
The original code is sorta funny to begin with. I think preferring a region list over the AZ map makes a lot more sense, if it's available.